src/shared/translation/translationEnglish.json

```
@@ -1575,5 +1575,91 @@
+        "Python-backend-development",
+        "Communication between frontend and backend using API endpoints with persistence in a Postgres database",
+        "Branch-focused development with review process on GitHub",
+        "Code based inline source documentation in the form of comments in frontend and backend"
+      ]
+    },
+    {
+      "header": "Conceptualization the system",
+      "body": [
+        "Theoretical preliminary conceptualization of an adaptive learning system",
+        "Modeling the individual level of knowledge using a rating system",
+        "Modeling student characteristics such as learning style and learning strategy",
+        "Literature research on adaptive learning systems, learning analytics, Knowledge Tracing, rating systems, large language models und natural language processing",
+        "Conception of a tool for the evaluation of free text tasks using large language models and natural language processing"
+      ]
+    },
+    {
+      "header": "Generation of learning paths",
+      "body": [
+        "Evaluation algorithms for various questionnaires (Index of Learning Styles - ILS and Learning Strategies in Studies, shortened - LIST-K)",
+        "Design, implementation and application of AI algorithms (e.g. ant colony algorithm and genetic algorithm) to generate individualized learning paths in a multidimensional hypercube",
+        "Implementation and application of literature-based algorithms based on the learning style (e.g. according to Graf et al. 2010)",
+        "Collaboration between frontend, backend and Moodle for the automatic generation of learning paths for students"
+      ]
+    },
+    {
+      "header": "Evaluation and exchange",
+      "body": [
+        "Planning and implementation of triangulated (quantitative and qualitative) evaluations of the frontend and learning elements",
+        "Elaboration of the findings in the form of peer-reviewed contributions at scientific conferences",
+        "Dissemination work (design of flyers, posters and roll-ups)",
+        "Attendance at pertinent and technically relevant (international) conferences",
+        "Internal and external exchange at workshops and network meetings"
+      ]
+    }
+  ],
+  "pages.aboutus.introductionHeader": "Partcipants of the HASKI project",
+  "pages.aboutus.introductionBody": "HASKI is a joint project of the three Bavarian universities OTH Regensburg, Aschaffenburg UAS & Kempten UAS and comprises over 20, lecturers, scientific assistants, student assistants and people involved in the administration.",
+  "pages.aboutus.quotesHeader": "Motivation",
+  "pages.aboutus.quotesBody": [
+    {
+      "quote": "\"What is special about HASKI's adaptive learner model is that it adapts to the learner and not the other way around - as is usually the case. This puts the learner at the center of the learning process.\"",
+      "name": "Prof. Dr. Georg Hagel",
+      "description": "Project Lead, Kempten University"
```

Hab gesehen, du hast überall "UAS" geschrieben. Wäre zur Vereinheitlichung hier dann auch passender.
```suggestion
      "description": "Project Lead, Kempten UAS"
```